source/modules/MEI.gestural.xml

```diff
@@ -541,6 +541,23 @@
       <memberOf key="att.timestamp2.gestural"/>
     </classes>
   </classSpec>
+  <classSpec ident="att.ornamentAccid.ges" module="MEI.gestural" type="atts">
+    <desc xml:lang="en">Gestural accidentals associated with ornaments.</desc>
+    <attList>
+      <attDef ident="accidupper.ges" usage="opt">
```

Comment:
Perhaps `upperaccid.ges` -- would align with `accid.ges`?

Reply:
Are you concerned about the naming? Since the plain attribute is `accidupper` then `accidupper.ges` is inline on that side. I think this is more important.
<img width="1272" alt="image" src="https://user-images.githubusercontent.com/689412/193578362-ac429965-1f41-4adf-80f0-7d1814ea4160.png">

Reply:
OK, sure.